packages/commerce-sdk-react/src/components/StorefrontPreview/storefront-preview.tsx

```diff
@@ -67,19 +70,11 @@ export const StorefrontPreview = ({
 StorefrontPreview.propTypes = {
     children: PropTypes.node,
     enabled: PropTypes.bool,
-    // a custom prop type function to only require this prop if enabled is true.
-    getToken: function (props: any, propName: any, componentName: any) {
-        if (
-            props['enabled'] === true &&
-            (props[propName] === undefined || typeof props[propName] !== 'function')
-        ) {
-            return new Error(
-                `${String(propName)} is a required function for ${String(
-                    componentName
-                )} when enabled is true`
-            )
-        }
-    }
+    // A custom prop type function to only require this prop if enabled is true. Ultimately we would like
+    // to get to a place where both these props are simply optional and we will provide default implementations.
+    // This would make the API simpler to use.
+    getToken: CustomPropTypes.requiredFunctionWhenEnabled,
+    onContextChange: CustomPropTypes.requiredFunctionWhenEnabled
```

Comment:
```suggestion
    onContextChange: PropTypes.func
```
This is just a basic optional prop.